nise/yaml_generators/aws/generator.py

```diff
@@ -74,26 +74,49 @@ def generate_tags(key, config, prefix="", suffix="", dynamic=True):
     return [dicta(key=key, v=generate_name(config)) for key in keys]
 
 
+def generate_resource_id_and_tag(config, key):
+    if not config.get("id_labels"):
```

Comment:
Small style nit:  It's generally more readable to put the positive condition first rather than the negative one.

I'm not going to block the PR on changing this. It's just something to consider for the future.